data/mods/DinoMod/monstergroups/misc.json

```diff
@@ -108,5 +108,66 @@
       { "monster": "mon_zeinonychus", "weight": 25, "cost_multiplier": 30, "starts": 672, "pack_size": [ 2, 3 ] },
       { "monster": "mon_zeinonychus", "weight": 25, "cost_multiplier": 30, "starts": 2160, "pack_size": [ 2, 3 ] }
     ]
+  },
+  {
+    "type": "monstergroup",
+    "name": "GROUP_CAMP_HUNTING",
+    "//": "Special group with hardcoded reference in basecamp::hunting_results. Does not actually spawn anywhere, used solely to determine returns from camp hunting mission.",
+    "//1": "Can be overwritten by mods.",
```

Comment:
Did you intend to completely overwrite these monster groups so that only Dino's can be hunted? Or did you intend to use `"copy-from"` so dinos could be hunted in addition to the standard animals?

Same question for the other groups.

Reply:
Addition for sure